include/aws/common/task_scheduler.h

```diff
@@ -89,8 +89,16 @@ extern "C" {
 
     /**
      * Sequentially execute all tasks that are ready until either the queue is empty or no ready tasks are available.
+     * time before next task is the time in nanoseconds (based on the configured aws_task_scheduler_clock) before the
```

Comment:
This sentence starts with a lower case letter. It is also unclear. There is an argument to the function called "next_task_time" but nothing called "time before next task". Is this an out of date description?

Reply:
will fix.